packages/gatsby-docs/src/pages/setup/customer_setup.mdx

```diff
@@ -2,20 +2,26 @@
 title: Setting up the tools
 ---
 
-> **Apologies, the UI Upgrade Helper is temporarily unavailable during a transition from IBM hosting services.**
+> **Apologies, the UI Upgrade Helper is temporarily unavailable for download during a transition from IBM hosting services.**
+
+<InlineNotification>
+
+   **Note:** In Social Program Management 8.1, Carbon was updated from v10 to v11. In Carbon v11, the CSS class name prefix "bx" was changed to "cds". For example, `bx-cluster` was changed to `cds-cluster`. If you are upgrading to Social Program Management 8.1 or later and have custom code, you must review your CSS to see if you are affected.
```

Comment:
Hey instead of `bx-cluster`, can we update this to be `bx--cluster`. Also can we update `cds-cluster` to be `cds--cluster`?